kiwi/system/setup.py

```diff
@@ -629,6 +635,12 @@ def create_fstab(self, entries):
                     fstab.write(append.read())
                 Path.wipe(fstab_append_file)
 
+        if os.path.exists(fstab_patch_file):
+            Command.run(
+                ['patch', '-i', fstab_patch_file]
+            )
+            Path.wipe(fstab_patch_file)
```

Comment:
A patch can fail. Do makes sense to detect it and abort?

Reply:
yes and if it fails the Command.run raises with an exception including the failed text from patch and this would be the end result of that run. It would end with a KiwiCommandError(...fail info from path). I considered this enough and did not catch it here for an extra validation

Do you think we need more ?

Reply:
> Do you think we need more ?

No! Is perfect :+1: